software/bsg_manycore_lib/bsg_set_tile_x_y.h

```diff
@@ -1,3 +1,21 @@
+/* Each application launched onto the manycore is represented by a grid of tile groups.        */
+/* A grid is a two diemsnional structure represeting the entire application, with each         */
+/* element of grid being a tile group with X and Y coordinates bsg_tile_group_id_x/y,          */
+/* and the flat coordiante of bsg_tile_group_id, which is calculate using grid dimensions      */
+/* __bsg_x               --> The X coordinate of tile inside its tile group                    */
+/* __bsg_y               --> The Y coordiante of tile inside its tile group                    */
+/* __bsg_id              --> The flat index of tile inside tile group                          */
+/*                            __bsg_id = __bsg_y * __bsg_tiles_X + __bsg_x                     */
```

Comment:
would be nice to add that __bsg_tiles_X is the x-dimension of the tile-group. also for __bsg_tiles_Y.